triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -198,12 +198,12 @@ object Converter {
         for {
```

Comment:
Can you please add a comment to the type definitions in `triggers/daml/Daml/Trigger/LowLevel.daml` that these numbers will have to be updated if you change the order of the constructors, thanks! (arguably that code should be rewritten to not require hardcoding them here but let’s not do this in this PR).

Reply:
Good point.